app/components/clients_carousel/index.jsx

```diff
@@ -1,35 +1,31 @@
 "use client";
 
 import React, { useEffect, useRef, useState } from "react";
-import { Autoplay, Pagination } from "swiper/modules";
+import { Autoplay } from "swiper/modules";
 import { Swiper, SwiperSlide } from "swiper/react";
 
 function Clients_Carousel({ data }) {
   return (
     <section id="clients">
-      <div className="w-full h-full">
-        <div className="w-full  bg-gray-100">
+      <div className="w-full h-full bg-white drop-shadow-md pt-8 pl-8 rounded-md overflow-hidden">
+        <div className="w-full ">
           <div>
-            <h1 className="relative p-8 text-3xl font-extrabold text-black text-center">
+            <h1 className="relative p-8 mb-2 leading-8 text-4xl font-extrabold  text-center tracking-tight underline underline-offset-8 decoration-4 decoration-indigo-600">
               Confian en Nosotros
             </h1>
           </div>
-          <div>
+          <div class="relative origin-center mySwiper">
             <Swiper
+              freeMode={true}
               loop={true}
-              speed={2000} // ( ej 2000 ms = 2 segundos)
+              speed={3000}
               autoplay={{
-                delay: 3000, // tiempo entre transiciones
-                disableOnInteraction: true, //inact en interaccion
-              }}
-              spaceBetween={0}
-              //centeredSlides={true}
-              pagination={{
-                type: "bullets",
-                dynamicBullets: "true",
-                dynamicMainBullets: "3",
+                delay: 0,
+                disableOnInteraction: true,
```

Comment:
we should add 
pauseOnMouseEnter | boolean | false | When enabled autoplay will be paused on pointer (mouse) enter over Swiper container.
-- | -- | -- | --


[pauseOnMouseEnter](https://swiperjs.com/swiper-api#param-pauseOnMouseEnter)	boolean	false	
When enabled autoplay will be paused on pointer (mouse) enter over Swiper container.